man/usr/share/man/man1m/piadm.1m.md

```diff
@@ -69,6 +69,11 @@ piadm(1M) -- Manage SmartOS Platform Images
     itself without issue.  Occasionally, however, a PI will have Boot Image
     changes also that need to accompany it.
 
+    Boot images can be customized by providing loader.conf.local and/or
+    loader.rc.local files in POOL/boot/common/. When a new boot image is
+    installed using `piadm install` these files are symlinked into the
+    images.
```

Comment:
To make this more clear, I think this paragraph should be changed to read as follows:

> The behavior of loader can be controlled by providing loader.conf.local and/or loader.rc.local files in the ${BOOTPOOL}/boot-${VERSION} directory. Loader config files can also be placed in ${BOOTPOOL}/common, and will be used by all subsequently installed boot images. See loader.conf(4) and loader(5) for the format of these files.